src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import { FC, useEffect } from 'react';
 
 import './App.css';
-import CardList from './components/CardList';
-import { useLocalStorage } from './hooks/useLocalStorage';
-import { CardService } from './services/card';
-import { client } from './services/client';
-import { CardListItemProps } from './components/CardList.types';
+import CardList from '@components/CardList';
```

Comment:
usually this annotation is used for namespaces and I do not see the proper implementation is place also do not see a usecase. Let's discuss this at a bit later point?